### PR TITLE
[DOCU-1567] issue template updates

### PR DESCRIPTION
**Pull request by falondarville**
### Review

@Kong/team-docs 

### Summary

* Update front matter
* Update form

### Reason

* Improve community ease of contribution
* Be more specific to increase clear communication
* Refer contributors to troubleshooting community

### Testing

Local testing.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
 <!-- 
-Did you spot a typo or do you want to contribute to documentation? Instead of a GitHub issue, we encourage you to submit a pull-request (https://help.github.com/articles/creating-a-pull-request/) and we will review your changes!
+Thank you for your interest in contributing!
 
-For all other issues, please fill in the sections below.
+Before submitting an issue, check our existing issues and pull requests to see if someone else has reported your issue previously. 
 
-Check existing issues and pull-requests before submitting new ones, as a courtesy to the maintainers and making sure work isn't duplicated.
+If you know how to resolve the issue you're reporting, we highly encourage that you submit a pull request instead: https://help.github.com/articles/creating-a-pull-request/.
```
Comment (Bradamant3) [inline .github/ISSUE_TEMPLATE.md:6]:
```suggestion
If you know how to resolve the issue you're reporting, we encourage you to submit a pull request instead: https://help.github.com/articles/creating-a-pull-request/.
```

Thinking that if we want to discourage adverbs (in style guide) we could lead by example 😀

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
+
+<!-- Link the part of the site that needs attention.  -->
 
 ### Summary
 
-SUMMARY_GOES_HERE
+<!-- 
+    Describe the issue in detail and, if applicable, tell us why this change is important.  
+        * Bad: "The ACL plugin documentation needs updating."
+        * Good: "Add the missing DELETE endpoint to ACL plugin documentation."
```
Comment (Bradamant3) [inline .github/ISSUE_TEMPLATE.md:20]:
```suggestion
        * Do: "Add the missing DELETE endpoint to ACL plugin documentation."
        * Don't: "The ACL plugin documentation needs updating."
```

Following the pattern we've got in the style guide already, plus it sounds more encouraging.

**Review by Bradamant3 — COMMENTED** [review]
Such nice work! A couple of suggestions only in service to expanding the welcome-and-lead-by-example approach you're already taking. ❤️

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
-Did you spot a typo or do you want to contribute to documentation? Instead of a GitHub issue, we encourage you to submit a pull-request (https://help.github.com/articles/creating-a-pull-request/) and we will review your changes!
+Thank you for your interest in contributing!
 
-For all other issues, please fill in the sections below.
+Before submitting an issue, check our existing issues and pull requests to see if someone else has reported your issue previously. 
 
-Check existing issues and pull-requests before submitting new ones, as a courtesy to the maintainers and making sure work isn't duplicated.
+If you know how to resolve the issue you're reporting, we encourage you to submit a pull request instead: https://help.github.com/articles/creating-a-pull-request/.
+
+If you have a troubleshooting issue, reach out to our community: https://github.com/Kong/kong/discussions.
```
Comment (lena-larionova) [inline .github/ISSUE_TEMPLATE.md:8]: Not sure how to phrase this, but I think this sentence needs to be a bit clearer - what exactly is a "troubleshooting issue"? Maybe make a point of drawing a contrast between code/product/etc troubleshooting vs something wrong with docs. 

E.g.: "If you need help troubleshooting issues with (your implementation? code? I don't know what phrase/word to use here), reach out to our community"

**Review by lena-larionova — COMMENTED** [review]
I just have one clarification, otherwise this looks great. Much more helpful!

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (falondarville) [inline .github/ISSUE_TEMPLATE.md:8]: "If you have a code-related troubleshooting issue, please reach out to our community." Thoughts?

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
-Did you spot a typo or do you want to contribute to documentation? Instead of a GitHub issue, we encourage you to submit a pull-request (https://help.github.com/articles/creating-a-pull-request/) and we will review your changes!
+Thank you for your interest in contributing!
 
-For all other issues, please fill in the sections below.
+Before submitting an issue, check our existing issues and pull requests to see if someone else has reported your issue previously. 
 
-Check existing issues and pull-requests before submitting new ones, as a courtesy to the maintainers and making sure work isn't duplicated.
+If you know how to resolve the issue you're reporting, we encourage you to submit a pull request instead: https://help.github.com/articles/creating-a-pull-request/.
+
+If you have a code-related troubleshooting issue, please reach out to our community.: https://github.com/Kong/kong/discussions.
```
Comment (lena-larionova) [inline .github/ISSUE_TEMPLATE.md:8]:
```suggestion
If you have a code-related troubleshooting issue, please reach out to our community: https://github.com/Kong/kong/discussions.
```

**Review by lena-larionova — APPROVED** [review]
Tiny typo, the smallest of deals. LGTM!

**Pull request merged**